cmd/link/ocmrole/cmd.go

```diff
@@ -127,8 +128,10 @@ func run(cmd *cobra.Command, argv []string) (err error) {
 		os.Exit(1)
 	}
 
-	if !confirm.Prompt(true, "Link the '%s' role with organization '%s'?", roleArn, orgAccount) {
-		os.Exit(0)
+	if cmd.Flags().Lookup("yes").Value.String() == "false" {
```

Comment:
I dont think this change is needed. It is implicitly handled in `confirm.Prompt`